clients/src/main/resources/common/message/ElectPreferredLeadersResponse.json

```diff
@@ -0,0 +1,39 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "apiKey": 43,
+  "type": "response",
+  "name": "ElectPreferredLeadersResponse",
+  "validVersions": "0",
+  "fields": [
+    { "name": "ThrottleTimeMs", "type": "int32", "versions": "0+",
+      "about": "The duration in milliseconds for which the request was throttled due to a quota violation, or zero if the request did not violate any quota." },
+    { "name": "ReplicaElectionResult", "type": "[]ReplicaElectionResult", "versions": "0+",
+      "about": "The error code, or 0 if there was no error.", "fields": [
+      { "name": "Topic", "type": "string", "versions": "0+",
+        "about": "The topic name" },
+      { "name": "PartitionResult", "type": "[]PartitionResult", "versions": "0+",
+        "about": "The topic name", "fields": [
```

Comment:
The topic name => partition ids?

Reply:
It's an array of structs including the id but also the error, so how about "The results for each partition".

Reply:
Sounds good.